src/components/minimap-toolbar/minimap-toolbar.scss

```diff
@@ -1,6 +1,14 @@
 @import '../../styles/variables';
 @import '../../styles/extends';
 
+.kui-theme--light {
+  --minimap-toolbar-active: $color-bg-light-3;
```

Comment:
Will this be needed for each component with custom colours?

Reply:
yes I tried to put them in each specific component so that if that component were deleted then the custom property would be deleted also.

but also AHHH i forgot to escape the variable! fixed.

Reply:
Ah makes sense. I think my comment about automating this with a little function would be reusable here as well, to make this a bit more fool proof and explicit.